Venmo/src/main/java/com/braintreepayments/api/VenmoPaymentAuthRequest.kt

```diff
@@ -0,0 +1,17 @@
+package com.braintreepayments.api
+
+/**
+ * A request used to launch the Venmo app for continuation of the Venmo payment flow.
+ */
+sealed class VenmoPaymentAuthRequest {
+
+    /**
+     * The request was successfully created and is ready to be launched by [VenmoLauncher]
+     */
+    class ReadyToLaunch(val requestParams: VenmoPaymentAuthRequestParams) : VenmoPaymentAuthRequest()
+
+    /**
+     * There was an [error] creating the request
+     */
+    class Failure(val error: Exception) : VenmoPaymentAuthRequest()
```

Comment:
I'm not sure if `Failure` feels right for returning a `Request` object - it might give the impression that the request itself failed, but in this case it's that the creating of the request failed. Is this naming pattern clear or is there a better way?